plugins/community/pseudomuto-doc/v1.5.1/Dockerfile

```
@@ -0,0 +1,11 @@
+# syntax=docker/dockerfile:1.4
+FROM golang:1.19.2-bullseye AS build
+RUN --mount=type=cache,target=/go/pkg/mod \
+    CGO_ENABLED=0 \
+    go install -ldflags "-s -w" -trimpath github.com/pseudomuto/protoc-gen-doc/cmd/protoc-gen-doc@1.5.1
```

```suggestion
    go install -ldflags "-s -w" -trimpath github.com/pseudomuto/protoc-gen-doc/cmd/protoc-gen-doc@v1.5.1
```

In order to build, this needs the leading `v` for the tag to be specified.